.travis.yml

```diff
@@ -17,6 +17,9 @@ cache:
         - $HOME/.stack
 
 matrix:
+  allow_failures:
+    - env: BUILD=qa
```

Comment:
Awesome. Pushing a fixup to test that one out (commenting out other envs to get quick feedback).